code/modules/clothing/clothing.dm

```diff
@@ -467,26 +476,31 @@
 	icon = 'icons/obj/clothing/shoes.dmi'
 	desc = "Comfortable-looking shoes."
 	gender = PLURAL //Carn: for grammatically correct text-parsing
+
+	body_parts_covered = FEET
+	slot_flags = SLOT_FLAG_FEET
+	dyeing_key = DYE_REGISTRY_SHOES
+
+	permeability_coefficient = 0.50
+	slowdown = SHOES_SLOWDOWN
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/shoes.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/shoes.dmi'
+		)
+
```

Comment:
you didn't include `dyeable = FALSE` here like in other places, but its false by default. Do you want to put a `dyeable = TRUE` here?

Reply:
updated in next commit